client/src/AddFriend.js

```diff
@@ -97,14 +103,40 @@ class AddFriend extends React.Component {
             onChange={this.handleChange}
             placeholder='Friends email'
           />
-          <input type='submit' value='Add Friend' />
+          {!this.state.isloading && submitButton()}
+          {this.state.isloading && loadButton()}
```

Comment:
To make this code cleaner, you can use a "ternary operator" which is basically a shortcut if/else statement


```
{ this.state.isLoading ?  loadButton() : submitButton() }
// same as 
// "if (this.state.isLoading) loadButton(); else submitButton()"

```